openedx/features/qverse_features/registration/models.py

```diff
@@ -79,15 +82,17 @@ class QVerseUserProfile(models.Model):
     This model is also used to add custom registration fields.
     """
     class Meta:
+        app_label = APP_LABEL
         verbose_name_plural = 'Qverse User Profiles'
 
     user = models.OneToOneField(User, unique=True, db_index=True,
                                 related_name='qverse_profile', on_delete=models.CASCADE)
-    department = models.ForeignKey(Department, db_index=True ,null=True, on_delete=models.SET_NULL)
+    department = models.ForeignKey(Department, db_index=True, null=True, on_delete=models.SET_NULL)
     registration_number = models.CharField(unique=True, max_length=30, db_index=True)
     other_name = models.CharField(blank=True, max_length=50)
     mobile_number = models.CharField(blank=True, max_length=20)
-    current_level = models.IntegerField(choices=LEVEL_CHOICES, validators=[validate_current_level])
+    current_level = models.IntegerField(choices=LEVEL_CHOICES,
+                                        validators=[validate_current_level, MaxValueValidator(99)])
```

Comment:
```suggestion
                                        validators=[validate_current_level, MaxValueValidator(9)])
```

Reply:
![image](https://user-images.githubusercontent.com/42294172/77164279-ba07bd80-6ad1-11ea-84de-2550957dc149.png)
Is the requirement provided from client.

Reply:
Okay